### PR TITLE
osfv_cli/src/osfv: Fix success message on flashing fail

### PR DESCRIPTION
**Pull request by SebastianCzapla**

### CODE REVIEW

**Comment by SebastianCzapla** [comment]
Fix for: https://github.com/Dasharo/osfv-scripts/issues/41

File: osfv_cli/src/osfv/cli/cli.py [hunk 1]
```diff
@@ -257,8 +257,11 @@ def flash_read(rte, args):
 
 def flash_write(rte, args):
     print(f"Writing {args.rom} to flash...")
-    rte.flash_write(args.rom)
-    print(f"Flash written")
+    rc = rte.flash_write(args.rom)
+    if rc == 0:
+        print(f"Flash written")
```
Comment (miczyg1) [inline osfv_cli/src/osfv/cli/cli.py:262]:
```suggestion
        print(f"Flash written successfully")
```

File: osfv_cli/src/osfv/rf/rte_robot.py [hunk 2]
```diff
@@ -76,7 +76,10 @@ def rte_flash_write(self, fw_file):
         """Writes file from ``fw_file`` path into DUT flash chip"""
         robot.api.logger.info(f"Writing {fw_file} to flash...")
         rc = self.rte.flash_write(fw_file)
-        robot.api.logger.info(f"Flash written")
+        if rc == 0:
+            robot.api.logger.info(f"Flash written")
```
Comment (miczyg1) [inline osfv_cli/src/osfv/rf/rte_robot.py:80]:
```suggestion
            robot.api.logger.info(f"Flash written successfully")
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]
Just to be precise

File: osfv_cli/src/osfv/rf/rte_robot.py (hunk 2 above)
Comment (SebastianCzapla) [reply osfv_cli/src/osfv/rf/rte_robot.py]: Changed commit to match suggested messages.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**